### PR TITLE
r/aws_lb_listener: update tags after creation

### LINKED ISSUE #19888: Default tags causes issues on creation of load balancer listener
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* aws_lb_listener

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
provider "aws" {
  default_tags {
    tags = {
      test  = "variable"
    }
  }
}


resource "aws_lb" "gateway" {
  name               = "gw-lb"
  internal           = false
  load_balancer_type = "gateway"
  subnets            = var.public_subnets.*.id
}

resource "aws_lb_listener" "gateway" {
  load_balancer_arn = aws_lb.gateway.arn
  tags_all          = {}

  default_action {
    type             = "forward"
    target_group_arn = aws_lb_target_group.gateway.arn
  }
}

```

### Debug Output

```ccl
  # aws_lb_listener.gateway will be created
  + resource "aws_lb_listener" "gateway" {
      + arn               = (known after apply)
      + id                = (known after apply)
      + load_balancer_arn = "arn:aws:elasticloadbalancing:xxx:xxxx:loadbalancer/gwy/gw-lb/75e4a91cbbe26624"
      + protocol          = (known after apply)
      + ssl_policy        = (known after apply)
      + tags_all          = {
          + test  = "variable"
        }

      + default_action {
          + order            = (known after apply)
          + target_group_arn = "arn:aws:elasticloadbalancing:xxx:xxxx:loadbalancer/gwy/gw-lb/75e4a91cbbe26624"
          + type             = "forward"
        }
    }

```
### Expected Behavior
Creation

### Actual Behavior
```
error creating ELBv2 Listener (arn:aws-us-gov:elasticloadbalancing:xxxx:xxx:loadbalancer/gwy/gw-lb/75e4a91cbbe26624): ValidationError: You cannot specify tags on creation of a Gateway Load Balancer listener
│ 	status code: 400, request id: c82b732b-422d-438d-acad-bf02c685839a
```

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

### LINKED ISSUE #20144: aws_lb_target_group - ValidationError: You cannot specify tags on creation of a GENEVE target group
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

0.14.11
AWS Provider 3.49.0

### Affected Resource(s)

* aws_lb_target_group

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
resource "aws_vpc" "security" {
  cidr_block           = "192.168.1.0/24"
  enable_dns_support   = true
  enable_dns_hostnames = true

  tags = {Name = "aws_lb_target_group_test_case"}
}



resource "aws_lb_target_group" "gwlb" {
  name                 = "gwlb"
  port                 = 6081
  protocol             = "GENEVE"
  vpc_id               = aws_vpc.security.id
  target_type          = "instance"
  deregistration_delay = "20"
  health_check {
    interval = 10
    port     = 80
    protocol = "TCP"
  }
  tags = {Name = "aws_lb_target_group_test_case"}
}


```

### Expected Behavior

Resource should be created successfully as it was applied before.

### Actual Behavior

Error Message:
```

       Error: error creating LB Target Group: ValidationError: You cannot specify tags on creation of a GENEVE target group
        status code: 400, request id: dddddddddddddddddddddddddddddd

```

I didn't get the error when using provider version `3.48.0`.

### Steps to Reproduce

1. `terraform apply`

### References

* #https://github.com/hashicorp/terraform-provider-aws/pull/20079 - This PR added default_tags handling. IMHO it's the root cause: https://github.com/hashicorp/terraform-provider-aws/commit/b0a239dcbcd45cc606e9aaa343674e82a62477ba

### PR DESCRIPTION
**Pull request by johnsonaj**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes #19888
Closes #20144

Output from acceptance testing:

```console
$ make testacc TESTS=TestAccELBV2Listener_ PKG=elbv2

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/elbv2/... -v -count 1 -parallel 20 -run='TestAccELBV2Listener_'  -timeout 180m
=== RUN   TestAccELBV2Listener_basic
=== PAUSE TestAccELBV2Listener_basic
=== RUN   TestAccELBV2Listener_tags
=== PAUSE TestAccELBV2Listener_tags
=== RUN   TestAccELBV2Listener_forwardWeighted
=== PAUSE TestAccELBV2Listener_forwardWeighted
=== RUN   TestAccELBV2Listener_Protocol_upd
=== PAUSE TestAccELBV2Listener_Protocol_upd
=== RUN   TestAccELBV2Listener_backwardsCompatibility
=== PAUSE TestAccELBV2Listener_backwardsCompatibility
=== RUN   TestAccELBV2Listener_Protocol_https
=== PAUSE TestAccELBV2Listener_Protocol_https
=== RUN   TestAccELBV2Listener_LoadBalancerARN_gatewayLoadBalancer
=== PAUSE TestAccELBV2Listener_LoadBalancerARN_gatewayLoadBalancer
=== RUN   TestAccELBV2Listener_Protocol_tls
=== PAUSE TestAccELBV2Listener_Protocol_tls
=== RUN   TestAccELBV2Listener_redirect
=== PAUSE TestAccELBV2Listener_redirect
=== RUN   TestAccELBV2Listener_fixedResponse
=== PAUSE TestAccELBV2Listener_fixedResponse
=== RUN   TestAccELBV2Listener_cognito
=== PAUSE TestAccELBV2Listener_cognito
=== RUN   TestAccELBV2Listener_oidc
=== PAUSE TestAccELBV2Listener_oidc
=== RUN   TestAccELBV2Listener_DefaultAction_order
=== PAUSE TestAccELBV2Listener_DefaultAction_order
=== RUN   TestAccELBV2Listener_DefaultAction_orderRecreates
=== PAUSE TestAccELBV2Listener_DefaultAction_orderRecreates
=== CONT  TestAccELBV2Listener_basic
=== CONT  TestAccELBV2Listener_Protocol_tls
=== CONT  TestAccELBV2Listener_oidc
=== CONT  TestAccELBV2Listener_fixedResponse
=== CONT  TestAccELBV2Listener_DefaultAction_orderRecreates
=== CONT  TestAccELBV2Listener_DefaultAction_order
=== CONT  TestAccELBV2Listener_cognito
=== CONT  TestAccELBV2Listener_backwardsCompatibility
=== CONT  TestAccELBV2Listener_LoadBalancerARN_gatewayLoadBalancer
=== CONT  TestAccELBV2Listener_Protocol_https
=== CONT  TestAccELBV2Listener_forwardWeighted
=== CONT  TestAccELBV2Listener_Protocol_upd
=== CONT  TestAccELBV2Listener_tags
=== CONT  TestAccELBV2Listener_redirect
--- PASS: TestAccELBV2Listener_LoadBalancerARN_gatewayLoadBalancer (102.33s)
--- PASS: TestAccELBV2Listener_DefaultAction_orderRecreates (172.66s)
--- PASS: TestAccELBV2Listener_oidc (184.65s)
--- PASS: TestAccELBV2Listener_DefaultAction_order (195.73s)
--- PASS: TestAccELBV2Listener_redirect (209.74s)
--- PASS: TestAccELBV2Listener_basic (215.41s)
--- PASS: TestAccELBV2Listener_Protocol_https (216.52s)
--- PASS: TestAccELBV2Listener_cognito (219.89s)
--- PASS: TestAccELBV2Listener_tags (223.28s)
--- PASS: TestAccELBV2Listener_Protocol_upd (236.04s)
--- PASS: TestAccELBV2Listener_forwardWeighted (239.78s)
--- PASS: TestAccELBV2Listener_backwardsCompatibility (244.94s)
--- PASS: TestAccELBV2Listener_fixedResponse (252.21s)
--- PASS: TestAccELBV2Listener_Protocol_tls (405.51s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2	408.312s
...
```

### CODE REVIEW

**Comment by johnsonaj** [comment]
```console
$ make testacc TESTS=TestAccELBV2TargetGroup_ PKG=elbv2

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/elbv2/... -v -count 1 -parallel 20 -run='TestAccELBV2TargetGroup_'  -timeout 180m
=== RUN   TestAccELBV2TargetGroup_backwardsCompatibility
=== PAUSE TestAccELBV2TargetGroup_backwardsCompatibility
=== RUN   TestAccELBV2TargetGroup_ProtocolVersion_basic
=== PAUSE TestAccELBV2TargetGroup_ProtocolVersion_basic
=== RUN   TestAccELBV2TargetGroup_ProtocolVersion_grpcHealthCheck
=== PAUSE TestAccELBV2TargetGroup_ProtocolVersion_grpcHealthCheck
=== RUN   TestAccELBV2TargetGroup_ProtocolVersion_grpcUpdate
=== PAUSE TestAccELBV2TargetGroup_ProtocolVersion_grpcUpdate
=== RUN   TestAccELBV2TargetGroup_HealthCheck_tcp
=== PAUSE TestAccELBV2TargetGroup_HealthCheck_tcp
=== RUN   TestAccELBV2TargetGroup_tls
=== PAUSE TestAccELBV2TargetGroup_tls
=== RUN   TestAccELBV2TargetGroup_HealthCheck_tcpHTTPS
=== PAUSE TestAccELBV2TargetGroup_HealthCheck_tcpHTTPS
=== RUN   TestAccELBV2TargetGroup_attrsOnCreate
=== PAUSE TestAccELBV2TargetGroup_attrsOnCreate
=== RUN   TestAccELBV2TargetGroup_basic
=== PAUSE TestAccELBV2TargetGroup_basic
=== RUN   TestAccELBV2TargetGroup_udp
=== PAUSE TestAccELBV2TargetGroup_udp
=== RUN   TestAccELBV2TargetGroup_ForceNew_name
=== PAUSE TestAccELBV2TargetGroup_ForceNew_name
=== RUN   TestAccELBV2TargetGroup_ForceNew_port
=== PAUSE TestAccELBV2TargetGroup_ForceNew_port
=== RUN   TestAccELBV2TargetGroup_ForceNew_protocol
=== PAUSE TestAccELBV2TargetGroup_ForceNew_protocol
=== RUN   TestAccELBV2TargetGroup_ForceNew_vpc
=== PAUSE TestAccELBV2TargetGroup_ForceNew_vpc
=== RUN   TestAccELBV2TargetGroup_Defaults_application
=== PAUSE TestAccELBV2TargetGroup_Defaults_application
=== RUN   TestAccELBV2TargetGroup_Defaults_network
=== PAUSE TestAccELBV2TargetGroup_Defaults_network
=== RUN   TestAccELBV2TargetGroup_HealthCheck_enable
=== PAUSE TestAccELBV2TargetGroup_HealthCheck_enable
=== RUN   TestAccELBV2TargetGroup_Name_generated
=== PAUSE TestAccELBV2TargetGroup_Name_generated
=== RUN   TestAccELBV2TargetGroup_Name_prefix
=== PAUSE TestAccELBV2TargetGroup_Name_prefix
=== RUN   TestAccELBV2TargetGroup_NetworkLB_targetGroup
=== PAUSE TestAccELBV2TargetGroup_NetworkLB_targetGroup
=== RUN   TestAccELBV2TargetGroup_networkLB_TargetGroupWithConnectionTermination
=== PAUSE TestAccELBV2TargetGroup_networkLB_TargetGroupWithConnectionTermination
=== RUN   TestAccELBV2TargetGroup_NetworkLB_targetGroupWithProxy
=== PAUSE TestAccELBV2TargetGroup_NetworkLB_targetGroupWithProxy
=== RUN   TestAccELBV2TargetGroup_preserveClientIPValid
=== PAUSE TestAccELBV2TargetGroup_preserveClientIPValid
=== RUN   TestAccELBV2TargetGroup_Geneve_basic
=== PAUSE TestAccELBV2TargetGroup_Geneve_basic
=== RUN   TestAccELBV2TargetGroup_Geneve_notSticky
=== PAUSE TestAccELBV2TargetGroup_Geneve_notSticky
=== RUN   TestAccELBV2TargetGroup_Stickiness_defaultALB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_defaultALB
=== RUN   TestAccELBV2TargetGroup_Stickiness_defaultNLB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_defaultNLB
=== RUN   TestAccELBV2TargetGroup_Stickiness_invalidALB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_invalidALB
=== RUN   TestAccELBV2TargetGroup_Stickiness_invalidNLB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_invalidNLB
=== RUN   TestAccELBV2TargetGroup_Stickiness_validALB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_validALB
=== RUN   TestAccELBV2TargetGroup_Stickiness_validNLB
=== PAUSE TestAccELBV2TargetGroup_Stickiness_validNLB
=== RUN   TestAccELBV2TargetGroup_tags
=== PAUSE TestAccELBV2TargetGroup_tags
=== RUN   TestAccELBV2TargetGroup_Stickiness_updateAppEnabled
=== PAUSE TestAccELBV2TargetGroup_Stickiness_updateAppEnabled
=== RUN   TestAccELBV2TargetGroup_HealthCheck_update
=== PAUSE TestAccELBV2TargetGroup_HealthCheck_update
=== RUN   TestAccELBV2TargetGroup_Stickiness_updateEnabled
=== PAUSE TestAccELBV2TargetGroup_Stickiness_updateEnabled
=== RUN   TestAccELBV2TargetGroup_HealthCheck_without
=== PAUSE TestAccELBV2TargetGroup_HealthCheck_without
=== RUN   TestAccELBV2TargetGroup_ALBAlias_basic
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_basic
=== RUN   TestAccELBV2TargetGroup_ALBAlias_changeNameForceNew
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_changeNameForceNew
=== RUN   TestAccELBV2TargetGroup_ALBAlias_changePortForceNew
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_changePortForceNew
=== RUN   TestAccELBV2TargetGroup_ALBAlias_changeProtocolForceNew
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_changeProtocolForceNew
=== RUN   TestAccELBV2TargetGroup_ALBAlias_changeVPCForceNew
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_changeVPCForceNew
=== RUN   TestAccELBV2TargetGroup_ALBAlias_generatedName
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_generatedName
=== RUN   TestAccELBV2TargetGroup_ALBAlias_lambda
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_lambda
=== RUN   TestAccELBV2TargetGroup_ALBAlias_lambdaMultiValueHeadersEnabled
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_lambdaMultiValueHeadersEnabled
=== RUN   TestAccELBV2TargetGroup_ALBAlias_missingPortProtocolVPC
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_missingPortProtocolVPC
=== RUN   TestAccELBV2TargetGroup_ALBAlias_namePrefix
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_namePrefix
=== RUN   TestAccELBV2TargetGroup_ALBAlias_setAndUpdateSlowStart
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_setAndUpdateSlowStart
=== RUN   TestAccELBV2TargetGroup_ALBAlias_tags
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_tags
=== RUN   TestAccELBV2TargetGroup_ALBAlias_updateHealthCheck
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_updateHealthCheck
=== RUN   TestAccELBV2TargetGroup_ALBAlias_updateLoadBalancingAlgorithmType
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_updateLoadBalancingAlgorithmType
=== RUN   TestAccELBV2TargetGroup_ALBAlias_updateStickinessEnabled
=== PAUSE TestAccELBV2TargetGroup_ALBAlias_updateStickinessEnabled
=== CONT  TestAccELBV2TargetGroup_backwardsCompatibility
=== CONT  TestAccELBV2TargetGroup_Stickiness_defaultNLB
=== CONT  TestAccELBV2TargetGroup_ALBAlias_updateStickinessEnabled
=== CONT  TestAccELBV2TargetGroup_ALBAlias_updateLoadBalancingAlgorithmType
=== CONT  TestAccELBV2TargetGroup_ALBAlias_updateHealthCheck
=== CONT  TestAccELBV2TargetGroup_ALBAlias_tags
=== CONT  TestAccELBV2TargetGroup_ALBAlias_setAndUpdateSlowStart
=== CONT  TestAccELBV2TargetGroup_ALBAlias_namePrefix
=== CONT  TestAccELBV2TargetGroup_ALBAlias_missingPortProtocolVPC
=== CONT  TestAccELBV2TargetGroup_ALBAlias_lambdaMultiValueHeadersEnabled
=== CONT  TestAccELBV2TargetGroup_ALBAlias_lambda
=== CONT  TestAccELBV2TargetGroup_ALBAlias_generatedName
=== CONT  TestAccELBV2TargetGroup_ALBAlias_changeVPCForceNew
=== CONT  TestAccELBV2TargetGroup_ALBAlias_changeProtocolForceNew
=== CONT  TestAccELBV2TargetGroup_ALBAlias_changePortForceNew
=== CONT  TestAccELBV2TargetGroup_ALBAlias_changeNameForceNew
=== CONT  TestAccELBV2TargetGroup_ALBAlias_basic
=== CONT  TestAccELBV2TargetGroup_HealthCheck_without
=== CONT  TestAccELBV2TargetGroup_Stickiness_updateEnabled
=== CONT  TestAccELBV2TargetGroup_HealthCheck_update
--- PASS: TestAccELBV2TargetGroup_ALBAlias_missingPortProtocolVPC (25.57s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_updateAppEnabled
--- PASS: TestAccELBV2TargetGroup_HealthCheck_without (26.41s)
=== CONT  TestAccELBV2TargetGroup_tags
--- PASS: TestAccELBV2TargetGroup_ALBAlias_lambda (28.07s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_validNLB
--- PASS: TestAccELBV2TargetGroup_backwardsCompatibility (28.61s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_validALB
--- PASS: TestAccELBV2TargetGroup_ALBAlias_namePrefix (28.90s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_invalidNLB
--- PASS: TestAccELBV2TargetGroup_ALBAlias_basic (33.26s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_invalidALB
--- PASS: TestAccELBV2TargetGroup_ALBAlias_generatedName (33.86s)
=== CONT  TestAccELBV2TargetGroup_ForceNew_vpc
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateHealthCheck (49.65s)
=== CONT  TestAccELBV2TargetGroup_Stickiness_defaultALB
=== CONT  TestAccELBV2TargetGroup_Geneve_notSticky
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeVPCForceNew (50.77s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_update (54.37s)
=== CONT  TestAccELBV2TargetGroup_Geneve_basic
--- PASS: TestAccELBV2TargetGroup_ALBAlias_lambdaMultiValueHeadersEnabled (54.52s)
=== CONT  TestAccELBV2TargetGroup_preserveClientIPValid
--- PASS: TestAccELBV2TargetGroup_ALBAlias_tags (55.40s)
=== CONT  TestAccELBV2TargetGroup_NetworkLB_targetGroupWithProxy
=== CONT  TestAccELBV2TargetGroup_networkLB_TargetGroupWithConnectionTermination
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeProtocolForceNew (56.39s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changePortForceNew (57.57s)
=== CONT  TestAccELBV2TargetGroup_NetworkLB_targetGroup
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeNameForceNew (61.04s)
=== CONT  TestAccELBV2TargetGroup_Name_prefix
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateStickinessEnabled (64.56s)
=== CONT  TestAccELBV2TargetGroup_Name_generated
--- PASS: TestAccELBV2TargetGroup_ALBAlias_setAndUpdateSlowStart (66.28s)
=== CONT  TestAccELBV2TargetGroup_HealthCheck_enable
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateLoadBalancingAlgorithmType (66.61s)
=== CONT  TestAccELBV2TargetGroup_Defaults_network
--- PASS: TestAccELBV2TargetGroup_Stickiness_invalidNLB (38.06s)
=== CONT  TestAccELBV2TargetGroup_Defaults_application
--- PASS: TestAccELBV2TargetGroup_Stickiness_defaultNLB (72.02s)
=== CONT  TestAccELBV2TargetGroup_attrsOnCreate
--- PASS: TestAccELBV2TargetGroup_Stickiness_updateEnabled (78.91s)
=== CONT  TestAccELBV2TargetGroup_ForceNew_protocol
--- PASS: TestAccELBV2TargetGroup_Stickiness_defaultALB (29.46s)
=== CONT  TestAccELBV2TargetGroup_ForceNew_port
--- PASS: TestAccELBV2TargetGroup_ForceNew_vpc (47.31s)
=== CONT  TestAccELBV2TargetGroup_ForceNew_name
=== CONT  TestAccELBV2TargetGroup_udp
--- PASS: TestAccELBV2TargetGroup_Stickiness_invalidALB (48.28s)
--- PASS: TestAccELBV2TargetGroup_Geneve_basic (30.27s)
=== CONT  TestAccELBV2TargetGroup_basic
--- PASS: TestAccELBV2TargetGroup_Name_prefix (23.60s)
=== CONT  TestAccELBV2TargetGroup_HealthCheck_tcp
--- PASS: TestAccELBV2TargetGroup_tags (58.41s)
=== CONT  TestAccELBV2TargetGroup_HealthCheck_tcpHTTPS
=== CONT  TestAccELBV2TargetGroup_tls
--- PASS: TestAccELBV2TargetGroup_Stickiness_validALB (62.80s)
--- PASS: TestAccELBV2TargetGroup_Defaults_application (24.65s)
=== CONT  TestAccELBV2TargetGroup_ProtocolVersion_grpcHealthCheck
--- PASS: TestAccELBV2TargetGroup_Name_generated (27.91s)
=== CONT  TestAccELBV2TargetGroup_ProtocolVersion_grpcUpdate
--- PASS: TestAccELBV2TargetGroup_preserveClientIPValid (38.49s)
=== CONT  TestAccELBV2TargetGroup_ProtocolVersion_basic
--- PASS: TestAccELBV2TargetGroup_Geneve_notSticky (43.28s)
--- PASS: TestAccELBV2TargetGroup_NetworkLB_targetGroupWithProxy (39.46s)
--- PASS: TestAccELBV2TargetGroup_networkLB_TargetGroupWithConnectionTermination (40.44s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_enable (35.63s)
--- PASS: TestAccELBV2TargetGroup_Defaults_network (36.04s)
--- PASS: TestAccELBV2TargetGroup_basic (19.02s)
--- PASS: TestAccELBV2TargetGroup_udp (24.22s)
--- PASS: TestAccELBV2TargetGroup_tls (20.08s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_grpcHealthCheck (20.08s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_updateAppEnabled (88.99s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_port (38.60s)
--- PASS: TestAccELBV2TargetGroup_attrsOnCreate (46.78s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_validNLB (91.35s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_basic (27.00s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_name (39.43s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_tcpHTTPS (37.06s)
--- PASS: TestAccELBV2TargetGroup_NetworkLB_targetGroup (64.62s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_tcp (40.13s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_grpcUpdate (34.89s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_protocol (48.51s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2	130.144s
```

File: internal/verify/verify.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ const (
 )
 
 func CheckISOErrorTagsUnsupported(partition string, err error) bool {
-	if partition == endpoints.AwsPartitionID {
+	if partition == endpoints.AwsIsoPartitionID {
```
Comment (YakDriver) [inline internal/verify/verify.go:57]: This needs to stay the way it was. In use, `true` is unsupported and bad. `false` means all good. If the partition is `aws`, all is good - none of the checks is needed - return false.

**Review by YakDriver — CHANGES_REQUESTED** [review]

File: internal/verify/verify.go (hunk 1 above)
Comment (johnsonaj) [reply internal/verify/verify.go]: Reverted back to original

**Review by YakDriver — APPROVED** [review]
Nice! 🎉 

```console
% make testacc TESTS=TestAccELBV2TargetGroup_ PKG=elbv2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/elbv2/... -v -count 1 -parallel 20 -run='TestAccELBV2TargetGroup_'  -timeout 180m
--- PASS: TestAccELBV2TargetGroup_Stickiness_defaultALB (106.10s)
--- PASS: TestAccELBV2TargetGroup_backwardsCompatibility (121.56s)
--- PASS: TestAccELBV2TargetGroup_Name_prefix (122.31s)
--- PASS: TestAccELBV2TargetGroup_Geneve_basic (142.90s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_enable (189.96s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_vpc (193.24s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_name (206.13s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_tcp (209.27s)
--- PASS: TestAccELBV2TargetGroup_networkLB_TargetGroupWithConnectionTermination (210.53s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeProtocolForceNew (210.85s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_port (211.33s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_tags (212.40s)
--- PASS: TestAccELBV2TargetGroup_ForceNew_protocol (213.08s)
--- PASS: TestAccELBV2TargetGroup_preserveClientIPValid (213.34s)
--- PASS: TestAccELBV2TargetGroup_attrsOnCreate (214.64s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateHealthCheck (216.21s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_namePrefix (110.34s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_missingPortProtocolVPC (136.11s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateLoadBalancingAlgorithmType (282.32s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_defaultNLB (297.09s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_updateStickinessEnabled (299.43s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_generatedName (106.19s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_setAndUpdateSlowStart (194.53s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_lambda (121.24s)
--- PASS: TestAccELBV2TargetGroup_udp (105.36s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_without (103.03s)
--- PASS: TestAccELBV2TargetGroup_NetworkLB_targetGroup (327.85s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_basic (118.51s)
--- PASS: TestAccELBV2TargetGroup_Name_generated (102.48s)
--- PASS: TestAccELBV2TargetGroup_Defaults_application (107.61s)
--- PASS: TestAccELBV2TargetGroup_Defaults_network (153.06s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeVPCForceNew (176.75s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_grpcHealthCheck (100.95s)
--- PASS: TestAccELBV2TargetGroup_tls (99.45s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_update (191.94s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changePortForceNew (195.58s)
--- PASS: TestAccELBV2TargetGroup_Geneve_notSticky (196.69s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_changeNameForceNew (195.90s)
--- PASS: TestAccELBV2TargetGroup_ALBAlias_lambdaMultiValueHeadersEnabled (275.16s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_basic (90.91s)
--- PASS: TestAccELBV2TargetGroup_basic (78.37s)
--- PASS: TestAccELBV2TargetGroup_ProtocolVersion_grpcUpdate (151.37s)
--- PASS: TestAccELBV2TargetGroup_NetworkLB_targetGroupWithProxy (153.21s)
--- PASS: TestAccELBV2TargetGroup_HealthCheck_tcpHTTPS (148.50s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_updateEnabled (243.76s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_validALB (133.32s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_invalidALB (136.76s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_invalidNLB (114.69s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_updateAppEnabled (177.92s)
--- PASS: TestAccELBV2TargetGroup_tags (175.16s)
--- PASS: TestAccELBV2TargetGroup_Stickiness_validNLB (219.23s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2	521.478s
% make testacc TESTS=TestAccELBV2Listener_ PKG=elbv2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/elbv2/... -v -count 1 -parallel 20 -run='TestAccELBV2Listener_'  -timeout 180m
--- PASS: TestAccELBV2Listener_LoadBalancerARN_gatewayLoadBalancer (145.74s)
--- PASS: TestAccELBV2Listener_DefaultAction_orderRecreates (210.16s)
--- PASS: TestAccELBV2Listener_backwardsCompatibility (212.02s)
--- PASS: TestAccELBV2Listener_cognito (220.29s)
--- PASS: TestAccELBV2Listener_basic (220.18s)
--- PASS: TestAccELBV2Listener_DefaultAction_order (223.18s)
--- PASS: TestAccELBV2Listener_Protocol_https (223.68s)
--- PASS: TestAccELBV2Listener_oidc (225.58s)
--- PASS: TestAccELBV2Listener_fixedResponse (226.26s)
--- PASS: TestAccELBV2Listener_redirect (233.75s)
--- PASS: TestAccELBV2Listener_Protocol_upd (264.34s)
--- PASS: TestAccELBV2Listener_tags (280.89s)
--- PASS: TestAccELBV2Listener_forwardWeighted (297.49s)
--- PASS: TestAccELBV2Listener_Protocol_tls (376.27s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/elbv2	378.099s
```

**Pull request merged**